Makefile

```diff
@@ -216,6 +216,17 @@ test-etcd: bin/confd bin/etcdctl bin/bird bin/bird6 bin/calico-node bin/kubectl
 		$(CALICO_BUILD) /tests/test_suite_etcd.sh
 	-git clean -fx etc/calico/confd
 
+.PHONY: ut
+## Run the fast set of unit tests in a container.
+ut: vendor
+	-mkdir -p .go-pkg-cache
+	docker run --rm -t --privileged --net=host \
+		-e LOCAL_USER_ID=$(LOCAL_USER_ID) \
+		-v $(CURDIR):/go/src/$(PACKAGE_NAME):rw \
+		-v $(CURDIR)/.go-pkg-cache:/go-cache/:rw \
+		-e GOCACHE=/go-cache \
+		$(CALICO_BUILD) sh -c 'cd /go/src/$(PACKAGE_NAME) && ginkgo -r --skipPackage vendor -skip "\[Datastore\]" $(GINKGO_ARGS) .'
```

Comment:
Wondering why we would skip "[Datastore]".  Perhaps that will become clearer in the rest of the review...

Reply:
I can't find "Datastore" in any of the code-managed files here, and removing the `-skip` seems to make no difference to the UT run, so I'll do that.  @briansan Please do let me know if I've got this wrong.

Reply:
@neiljerram Yep, this is just copy/paste from libcalico, where we _do_ have some tests marked as [Datastore]